engine/swarm/services.md

```diff
@@ -472,6 +473,29 @@ flag. For more information, see
 
 ### Control service scale and placement
 
+Swarm services provide a few different ways for you to control scale and
+placement of services on different nodes.
+
+- You can specify whether the service needs to run a specific number of replicas
+  or should run globally on every swarm node. See
+  [_replicated or global services](#replicated-or-global-services).
+
+- You can configure the service to run only on nodes with specific (arbitrary)
+  metadata set, and **cause the service to fail if appropriate nodes do not
+  exist**. This is referred to as
+  [_placement constraints_](#placement-constraints).
+
+- You can configure a replicated service to try to run only on nodes with
+  an arbitrary label set, but to **be able to run even if no appropriate nodes
+  are available**. This is referred to as
+  [_placement preferences_](#placement-preferences).
```

Comment:
That's not really what placement preferences are. They're about trying to balance tasks equally between groups of nodes with certain labels, for example a datacenter label. There is some technical documentation here: https://github.com/docker/swarmkit/blob/master/design/topology.md